validation/src/verifier/opts.rs

```diff
@@ -0,0 +1,22 @@
+#[derive(Clone, Debug)]
+pub struct VerifyOpts {
+    /// The point in time at which the capability must be valid, defaults to `now`
+    pub at_time: Option<chrono::DateTime<chrono::Utc>>,
+    /// How long the capability stays valid for after it was expired
+    pub revocation_phaseout_secs: chrono::Duration,
+    /// The clock tolerance when verifying iat, nbf, and exp
+    pub clock_skew: chrono::Duration,
+    /// Do not verify expiration time when false
+    pub check_exp: bool,
```

Comment:
remove this boolean and use at_time without a default (none being do not check exp)

Reply:
actually, this won't quite work since we use the time (now) to check the nbf/iat fields in addition to exp.. so leaving as is for now